test/gscraper/accounts_test.exs

```diff
@@ -0,0 +1,21 @@
+defmodule Gscraper.AccountsTest do
+  use Gscraper.DataCase
+
+  alias Gscraper.Accounts
+
+  describe "users" do
```

Comment:
To follow [our convention](https://nimblehq.co/compass/development/code-conventions/ex-unit/#formatting). Let's put a function name in describe block.
```elixir
describe "create_user/1" do
  test "creates a user when given valid params" do
  
  end

  test "returns error changeset when given invalid params" do
  
  end
end
```

Reply:
fixed in 7c36f2d.